experimental/terraform/tools/fourkeys-builder/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
 
-RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - && \
-    apt-get install software-properties-common -y && \
-    apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com $(lsb_release -cs) main" && \
-    apt update && \
-    apt install terraform=0.15.5
\ No newline at end of file
+ARG VERSION=1.0.2
```

Comment:
Do we need to set up CI/CD for this builder?

Reply:
IMHO, no? It's a one-off manual job  to publish that builder, and any future updates _should_ be pretty minor (bump TF, bump gcloud) and can be easily tested by running the e2e build.